index.bs

```diff
@@ -1137,6 +1137,8 @@ optional |expires|,
 |partitioned|
 run the following steps:
 
+1. Set |name| to |name|, [=stripped leading and trailing ASCII whitespace=].
+1. Set |value| to |value|, [=stripped leading and trailing ASCII whitespace=].
```

Comment:
This is wrong. Because you now you strip newlines and `\f` for which the next step would have returned failure. And we want to continue to return failure for those. You either have to do this after the next step or only strip U+0009 and U+0020.

Reply:
Fixed to only strip 0x9 and 0x20